lib/bolt/target.rb

```diff
@@ -129,6 +129,10 @@ def transport
       inventory_target.transport
     end
 
+    def transport_config
```

Comment:
This might also be a good opportunity to get rid of the `options` instance variable and alias, since those can't be defined in inventory or accessed by the target anymore.